### PR TITLE
Add Minidox keymap by dustypomerleau

### PR DESCRIPTION
**Pull request by dustypomerleau**
Add a Minidox keymap folder for dustypomerleau containing keymap, rules, config, and README.

### CODE REVIEW

File: keyboards/minidox/keymaps/dustypomerleau/keymap.c [hunk 1]
```diff
@@ -0,0 +1,490 @@
...
+#define SFT_7 LSFT_T(KC_7)
+#define SFT_A LSFT_T(KC_A)
+#define SFT_O LSFT_T(KC_O)
+#define SFT_OS OSM(MOD_LSFT)
+#define SFT_QOT LSFT_T(KC_QUOT)
+#define SYM_OS OSL(_SYM)
+#define SYS_Z LT(_SYS, KC_Z)
+#define VOL_DN S(LALT(KC__VOLDOWN))
+#define VOL_UP S(LALT(KC__VOLUP))
+#define XXXXXXX KC_NO
```
Comment (Rionlion100) [inline keyboards/minidox/keymaps/dustypomerleau/keymap.c]: Line 106 Not needed if you use QMK_KEYBOARD_H

File: keyboards/minidox/keymaps/dustypomerleau/config.h [hunk 2]
```diff
@@ -0,0 +1,30 @@
+// dustypomerleau, Minidox config
+
+#ifndef CONFIG_USER_H
```
Comment (drashna) [inline keyboards/minidox/keymaps/dustypomerleau/config.h]: If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`? 

It's simpler, and less prone to user error, so if you could make this switch? 

```suggestion
#pragma once
```

File: keyboards/minidox/keymaps/dustypomerleau/config.h [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+#define CONFIG_USER_H
```
Comment (drashna) [inline keyboards/minidox/keymaps/dustypomerleau/config.h]:
```suggestion
```

File: keyboards/minidox/keymaps/dustypomerleau/config.h [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+// #define MOUSEKEY_DELAY 0 //  delay before cursor movement (high feels sluggish, low makes fine movement difficult)
+// #define MOUSEKEY_INTERVAL 20 // time between movement reports - low settings feel like high mouse speed
+// #define MOUSEKEY_MAX_SPEED 10
+// #define MOUSEKEY_TIME_TO_MAX 60
+// #define MOUSEKEY_WHEEL_DELAY 0
+// #define MOUSEKEY_WHEEL_MAX_SPEED 8
+// #define MOUSEKEY_WHEEL_TIME_TO_MAX 4
+
+#endif
```
Comment (drashna) [inline keyboards/minidox/keymaps/dustypomerleau/config.h]:
```suggestion
```

File: keyboards/minidox/keymaps/dustypomerleau/config.h [hunk 5]
```diff
@@ -0,0 +1,30 @@
+// dustypomerleau, Minidox config
+
+#ifndef CONFIG_USER_H
+#define CONFIG_USER_H
+
+#include "config_common.h"
```
Comment (drashna) [inline keyboards/minidox/keymaps/dustypomerleau/config.h]: This line is not actually needed, and should be removed. 

Specifically, this should be included in the keyboard's `config.h` file, and is redundant. 

```suggestion
```

File: keyboards/minidox/keymaps/dustypomerleau/config.h [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+#define EE_HANDS
+#define IGNORE_MOD_TAP_INTERRUPT
+#define ONESHOT_TIMEOUT 1000
+#define PREVENT_STUCK_MODIFIERS
```
Comment (drashna) [inline keyboards/minidox/keymaps/dustypomerleau/config.h]: This is no longer needed, as it is now the default behavior.

File: keyboards/minidox/keymaps/dustypomerleau/keymap.c (hunk 1 above)
Comment (drashna) [reply keyboards/minidox/keymaps/dustypomerleau/keymap.c]: These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```

File: keyboards/minidox/keymaps/dustypomerleau/config.h (hunk 6 above)
Comment (dustypomerleau) [reply keyboards/minidox/keymaps/dustypomerleau/config.h]: Thanks for the input! All changes accepted and committed.

**Pull request merged**